yoga/Yoga.c

```diff
@@ -179,10 +179,26 @@ static inline float YGComputedEdgeValue(const float edges[YGEdgeCount],
   }
 
   if (edge == YGEdgeStart || edge == YGEdgeEnd) {
-    return YGUndefined;
+    YGValue result;
+    result.value = YGUndefined;
+    result.defined = false;
+    result.unit = YGUnitPixel;
+    return result;
   }
 
-  return defaultValue;
+  YGValue result;
+  result.value = defaultValue;
+  result.defined = (defaultValue < 0 || defaultValue >= 0); /* is faster than a nan function call and enough at this point */
+  result.unit = YGUnitPixel;
+  return result;
+}
+
+static inline float YGValueToFloat(const YGValue unit, const float parentSize) {
```

Comment:
`YGValueToFloat` is not very descriptive of what the function does. I would prefer `YGValueResolve` or something similar. What do you think?